### ISSUE TITLE
Property 'claimsSet' not found

### ISSUE THREAD

**Issue by dhiogocorrea** [zaguiini/react-native-pure-jwt#32]
Hello!

When I'm trying to build my app for ios, I'm getting this error:

![image](https://user-images.githubusercontent.com/10632653/120950412-17ab1880-c71d-11eb-8174-6a8b3e7695ac.png)

I already tried manual linking and it didn't work... any ideas?

**Comment by Mladen1kus** [comment]
Having the same issue, any solutions?

**Comment by Mladen1kus** [comment]
There seems to be a new release on https://github.com/yourkarma/JWT/releases

Had to manually set JWT to 
s.dependency 'JWT', '3.0.0-beta.13'
inside
https://github.com/zaguiini/react-native-pure-jwt/blob/master/react-native-pure-jwt.podspec
and reinstall the pods, 
@zaguiini, would you be able to at least accept pull request if we create one?

**Comment by aidanarmstrong** [comment]
i have this issue as well. Those changes you suggested do not work for me as i still get the claimSet error even went setting 'JWT', '3.0.0-beta.12' it reverts back to 11

**Comment by Mladen1kus** [comment]
That's strange, you can check in Podfile.lock which pod has jwt as dependency. Setting pod in your project podfile should override that, either 11 or 12 should work fine, 13 and 14 were both released in less than 24 hours. Remember to remove Pods and Podfile before doing pod install again, since Podile locks the versions.

**Comment by aidanarmstrong** [comment]
Ah my bad, i'm an idiot i was to focused on doing pod install haha yeah adding  pod 'JWT', '3.0.0-beta.12'to the pod file allowed it to be built successfully thanks man :)

**Comment by dhiogocorrea** [comment]
it worked guys, thanks! i'm not really into the project to fix this, but would be great to have this fix.

**Comment by odinsey** [comment]
Hi All,

Installing JWT (3.0.0-beta.14)
Installing react-native-pure-jwt (2.2.2)

and still have 
..../node_modules/react-native-pure-jwt/ios/RNPureJwt.m:53:43: Property 'claimsSet' not found on object of type 'JWTEncodingBuilder *'
..../node_modules/react-native-pure-jwt/ios/RNPureJwt.m:81:80: Property 'claimsSet' not found on object of type 'JWTDecodingBuilder *'
..../node_modules/react-native-pure-jwt/ios/RNPureJwt.m:88:54: Property 'claimsSet' not found on object of type 'JWTCodingResultTypeSuccess *'


As far as I now, it's not fixed as it's in the remap method (react-native side) that we access property claimSets that are not present.
You can update JWT pod as far as you want, if the remap method is not mirroring JWT pod it won't be ok.

**Comment by Mladen1kus** [comment]
@odinsey you're supposed to use any version that is not 13 or 14 and you should be ok for now (quickfix).

**Comment by AndrinGautschi** [comment]
It's very missleading to have this issue closed. Enforcing the version does solve the issue (at least for me) but it's far away from beeing resolved or how it should be. For future searchers:

Add the `pod 'JWT', '3.0.0-beta.12'` line to your Podfile like:

```
target 'MyApp' do
  ...
  pod 'JWT', '3.0.0-beta.12'
  ...
end
```

**Comment by saeedtkh** [comment]
@AndrinGautschi: It is working like a charm! Thanks a lot. :)

**Comment by Uttu316** [comment]
> It's very missleading to have this issue closed. Enforcing the version does solve the issue (at least for me) but it's far away from beeing resolved or how it should be. For future searchers:
> 
> Add the `pod 'JWT', '3.0.0-beta.12'` line to your Podfile like below, rerun `pod install` and verify the installed version in the Pofile.lock.
> 
> ```
> target 'MyApp' do
>   ...
>   pod 'JWT', '3.0.0-beta.12'
>   ...
> end
> ```

adding this line cause duplicate architecture error.

**Comment by dittmarconsulting** [comment]
@Uttu316 try this.

1. Delete Podfile.lock
2. `pod deintegrate`
3. `pod repo remove trunk`
4. `pod install --repo-update`

Good luck

**Comment by zaguiini** [comment]
Sorry for not paying much attention to this problem. It's certainly a showstopper.

It also makes me sad that a lot of people proposed solutions here, but no one took a few minutes of their time to create a pull request and make the package better. Anyway...

This is fixed and released on version `2.2.3`.
